### PR TITLE
More tests

### PR DESCRIPTION
**Pull request by dwalintukan**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,6 +19,10 @@ before_install:
 
 install:
   - yarn install
+  - npm install -g ethereumjs-testrpc
```
Comment (frankobe) [inline .travis.yml]: should we use either npm or yarn solely for package management

File: .travis.yml [hunk 2]
```diff
@@ -19,6 +19,10 @@ before_install:
...
+
+before_script:
+  - testrpc_high_value.sh
 
 script:
-  - npm test
\ No newline at end of file
+  - truffle test
```
Comment (frankobe) [inline .travis.yml]: nit. A blank line at EOF

File: test/BodhiToken_spec.js [hunk 3]
```diff
@@ -29,9 +36,13 @@ contract('BodhiToken', function(accounts) {
       let expectedSaleAmount = web3.toBigNumber(web3.toWei(60e6, "ether"));
       assert.equal(maxTokenForSale.toString(), expectedSaleAmount.toString(), "Sale amount does not match.");
 
-      let totalSupply = web3.toBigNumber(await token.tokenTotalSupply());
-      let expectedTotalSupply = web3.toBigNumber(web3.toWei(100e6, "ether"));
-      assert.equal(totalSupply.toString(), expectedTotalSupply.toString(), "Total token supply does not match.");
+      let totalTokenSupply = web3.toBigNumber(await token.tokenTotalSupply());
+      let expectedTotalTokenSupply = web3.toBigNumber(web3.toWei(100e6, "ether"));
+      assert.equal(totalTokenSupply.toString(), expectedTotalTokenSupply.toString(), "Total token supply does not match.");
```
Comment (frankobe) [inline test/BodhiToken_spec.js]: why compare after casting to str?

Comment (dwalintukan) [reply test/BodhiToken_spec.js]: the JS tests do not like to compare big numbers (they just fail for some reason). and Strings can handle any amount of number values (length).

File: .travis.yml (hunk 1 above)
Comment (dwalintukan) [reply .travis.yml]: Ah I was not aware that yarn can install every npm package. Will change.

File: .travis.yml (hunk 2 above)
Comment (dwalintukan) [reply .travis.yml]: Changed in other PR.

**Pull request merged**